src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,122 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+public class Storage {
+
+    private String filePath;
+
+    /**
+     * Constructor for Storage.
+     * @param filePath String of the file path leading to the data storage file
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+

```

Comment:
Not sure if the whitespace is here because you want to add anything else but remember to delete it after!